modules/customizing-the-appearance/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -1,129 +1,60 @@
-          priority: 100 # controls the order of this top-level menu item
-----
---
-
-. To ensure that a menu item is identified as a main menu item, you must add the `default.` prefix to its key. For example:
+        default._<menu_item_name>_: # <2>
+          parent: _<my_menu_group>_ # <3>
+          priority: 10 <4>
+        default._<menu_group_parent_item_name>_: # <4>
+          icon: _<my_menu_group_icon>_ # <5>
+          title: _<my_menu_group_title>_ # <6>
+          priority: 100 # <7>
+----
+<1> `default.main-menu-items`: Specifies the key for configuring static main menu items. The `default.` prefix identifies an item as a main menu item. You can add the `default.` prefix to both individual menu items or parent menu group configuration, such as `default.<menu_group_parent_item_name>` in the previous example.
+<2> `default._<menu_item_name>_`: Specifies the key of the menu item configuration. The `default.` prefix is required for a main menu item key.
+<3> `parent`: (Optional) Specifies the parent menu item for this item.
+<4> `priority`: (Optional) Specifies the order of this menu item within its menu level.
+<5> `icon`: Specifies the icon for the menu group. Required for parent menu items.
+<6> `title`: Specifies the title for the menu group. Required for parent menu items.
+<7> `priority`: (Optional) Specifies the order of the parent menu item in the sidebar.
 +
---
-.Example configuration of main menu items in sidebar navigation
-[source,yaml]
-----
-dynamicPlugins:
-  frontend:
-    default.main-menu-items: # key for configuring static main menu items
-      menuItems: 
-        default.<menu_item_name>: # key of the menu item configuration. `default.` prefix is required for a main menu item key <1>
-          parent: my_menu_group # optional, specifies the parent menu item for this item
-          priority: 10 # optional, specifies the order of this menu item within its menu level
-        default.<menu_group_parent_item_name>: # must be configured if it is specified as the parent of any menu items. `default.` prefix is required for a main group parent item key <1>
-          icon: my_menu_group_icon # required for parent menu items, defines the icon for the menu group
-          title: my_menu_group_title # required for parent menu items, defines the icon for the menu group
-          priority: 100 # optional, specifies the order of the parent menu item in the sidebar
-----
-
-
-<1> The `default.` prefix identifies an item as a main menu item. You can add the `default.` prefix to both individual menu items or parent menu group configuration, such as `default.<menu_group_parent_item_name>` in the previous example.
-
 [NOTE]
 ====
 The default priority of main menu items determines their order in the sidebar. You can customize the order of the static main menu items by adjusting their priority values. Ensure that the priority and title of each item is clear to facilitate easy reordering. 
-====
```

Comment:
Remove fluff. Consider removing the note which is repeating content already in the admonition.

For consistency consider rewriting the callout with leftover information still in the note, and more importantly with line 30 in mind (previous occurence of a `priority` key in this procedure), to help the user understand how similar or different this key is behaving in both contexts.